diff -ur hydrogen-0.9.0/lib/Hydrogen.cpp hydrogen-patch/lib/Hydrogen.cpp
--- hydrogen-0.9.0/lib/Hydrogen.cpp	2004-09-06 09:40:36.000000000 +0000
+++ hydrogen-patch/lib/Hydrogen.cpp	2004-10-13 12:41:59.085000728 +0000
@@ -2540,7 +2540,7 @@
 
 	float fBPM = 60000.0 / fInterval;
 
-	if ( fabs(fOldBpm1 - fBPM) > 20 ) {	// troppa differenza, niente media
+	if ( fabsf(fOldBpm1 - fBPM) > 20 ) {	// troppa differenza, niente media
 		fOldBpm1 = fBPM;
 		fOldBpm2 = fBPM;
 		fOldBpm3 = fBPM;
