--- hydrogen-0.9.4-orig/Sconstruct	2010-01-24 09:37:04.058943573 +0100
+++ hydrogen-0.9.4/Sconstruct	2010-01-24 09:59:11.132560775 +0100
@@ -54,7 +54,7 @@
 				cppflags.append(flag)
 			
 		else:
-			cppflags += ['-O3', '-fomit-frame-pointer', '-funroll-loops']
+			cppflags += ['-fomit-frame-pointer', '-funroll-loops']
 			#cppflags += " %s" % get_optimized_flags( target_cpu )
 			for flag in env["optflags"].split(" "):
 				cppflags.append(flag)
@@ -96,6 +96,9 @@
 	else:
 		raise Exception( "Platform '%s' not supported" % sys.platform )
 
+	# for flag in env["ldflags"].split(","):
+	#	ldflags.append(flag)
+
 	return (includes, cppflags, ldflags)
 
 
