diff -dur hydrogen-1.0.0-beta1.orig/CMakeLists.txt hydrogen-1.0.0-beta1/CMakeLists.txt
--- hydrogen-1.0.0-beta1.orig/CMakeLists.txt	2018-03-18 12:50:43.000000000 +0100
+++ hydrogen-1.0.0-beta1/CMakeLists.txt	2019-04-18 09:12:23.000000000 +0200
@@ -361,7 +361,7 @@
 	INSTALL(FILES ${CMAKE_SOURCE_DIR}/linux/hydrogen.appdata.xml DESTINATION "${CMAKE_INSTALL_PREFIX}/share/appdata")
 	INSTALL(FILES ${CMAKE_SOURCE_DIR}/linux/hydrogen.desktop DESTINATION "${CMAKE_INSTALL_PREFIX}/share/applications")
 	INSTALL(FILES ${CMAKE_SOURCE_DIR}/data/img/gray/h2-icon.svg DESTINATION "/usr/share/pixmaps")
-	INSTALL(FILES ${CMAKE_SOURCE_DIR}/linux/hydrogen.1 DESTINATION "${CMAKE_INSTALL_PREFIX}/man/man1")
+	INSTALL(FILES ${CMAKE_SOURCE_DIR}/linux/hydrogen.1 DESTINATION "${CMAKE_INSTALL_PREFIX}/share/man/man1")
 ENDIF()
 
 #
